views/showcase.html

```diff
@@ -164,6 +164,18 @@ <h2 class="bg-gray-200 px-4 py-3 font-bold mt-4 mb-1">{{__('Metadata')}}</h2>
             {{snippets.listify([dataset.license])}}
           </dd>
           {% endif %}
+          {% if dataset.groups %}
+          <dt class="info-list_title">Grupper</dt>
+          <dd class="info-list_description">
+            {% for group in dataset.groups %}
+            <p
+              <span class="colGroup break-words">
+                {{group.title}}
```

Comment:
I would expect this to be clickable so that I can go to a group's page.

Reply:
Now Group items are clickable. Click on item create query with that search term on all datasets.
![group_clickable](https://user-images.githubusercontent.com/12686547/71575269-a78e7100-2aec-11ea-8a95-351e050ab9ce.png)